### PR TITLE
Update to work with WebSocketApp 0.58.0

### PR DESCRIPTION
**Pull request by chucklay**

### CODE REVIEW

File: stomp_ws/client.py [hunk 1]
```diff
@@ -42,21 +42,21 @@ def _connect(self, timeout=0):
...
         self.opened = True
 
-    def _on_close(self):
+    def _on_close(self, ws):
         self.connected = False
         logging.debug("Whoops! Lost connection to " + self.ws.url)
         self._clean_up()
 
-    def _on_error(self, error):
+    def _on_error(self, ws):
```
Comment (rad-pat) [inline stomp_ws/client.py]: error not available on line below?

**Review by rad-pat — CHANGES_REQUESTED** [review]
Minor tweak

File: stomp_ws/client.py (hunk 1 above)
Comment (chucklay) [reply stomp_ws/client.py]: Ha, side effect of trying to fix this twice.

**Review by inviscid — APPROVED** [review]
LG after fix

File: stomp_ws/client.py (hunk 1 above)
Comment (inviscid) [reply stomp_ws/client.py]: was fixed

**Review by rad-pat — APPROVED** [review]

**Pull request merged**